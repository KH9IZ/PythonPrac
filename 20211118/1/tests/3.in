@objcount
class Sardina(str):
    def __init__(self, a):
        print(f"{a=}")

print(Sardina('1'))
print(Sardina.counter)
one = Sardina('010')
print(Sardina.counter)
