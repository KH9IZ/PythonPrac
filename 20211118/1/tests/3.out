a='1'
1
0
a='010'
1
